hs_core/page_processors.py

```diff
@@ -50,14 +51,29 @@ def get_page_context(page, user, resource_edit=False, extended_metadata_layout=N
 
     metadata_status = _get_metadata_status(content_model)
 
-    relevant_tools = []
-    for res_type in ToolResourceType.objects.all():
-        if str(content_model.content_model).lower() in str(res_type.tool_res_type).lower():
-            url = res_type.content_object.url_bases.first()
-            if url:
-                tl = {'title': res_type.content_object.title,
-                      'url': "{}{}{}".format(url.value, "/?res_id=", content_model.short_id)}
-                relevant_tools.append(tl)
+
+    relevant_tools = None
+    if not resource_edit: # view mode
+        relevant_tools = []
+        for res_type in SupportedResTypes.objects.all():
+            if str(content_model.content_model).lower() in str(res_type.get_supported_res_types_str()).lower():
```

Comment:
You should do:

    str(res_type.get_supported_res_types_str()).lower()

outside of the for loop.